js/explore/pack_hierarchy.js

```diff
@@ -210,11 +213,73 @@ function draw_pack_hierarchy(areaID) {
             childCircles.attr('r', d => d.r * k);
         }
 
+        // Data for legend
+        const labels = ['External Contributors', 'Internal Contributors', 'Repositories', 'GitHub Organizations', 'LLNL'];
+    
+        // Creates legend
+        const legend = chart
+            .append('g');
+
+        function updateLegend(labels, color = colors) {
+            legend.selectAll('g').remove();
+
+            const legendMap = [];
+            color.forEach((d, i) => {
+                legendMap.push({ text: labels[i], color: d });
+            });
+
+            legend.append('rect')
+                .attr('fill', '#FFFFFF')
+                .attr('fill-opacity', 0.7)
+                .attr('height', labels.length * (legendRectSize + legendSpacing) + legendSpacing)
+                .attr('width', 160)
+                .attr('y', 0 - legendSpacing)
+                .attr('x', -5)
+                .attr('rx', 10);
```

Comment:
The contents of the legend shouldn't be changing unless we change them, so I agree that a static box should be fine.

I have just a couple cosmetic change suggestions.
```suggestion
            legend.append('rect')
                .attr('fill', '#FFFFFF')
                .attr('fill-opacity', 0.9)
                .attr('height', labels.length * (legendRectSize + legendSpacing) + legendSpacing)
                .attr('width', 150)
                .attr('y', 0 - legendSpacing)
                .attr('x', -5)
                .attr('rx', 10);
```